docs/troubleshooting/untrusted-localhost-certificate.md

```diff
@@ -20,22 +20,23 @@ Additionally, there are warning messages from Kestrel written to the console tha
 
 ## Possible solutions
 
-Close all browser windows and try again. If you're still experiencing the issue, then attempt to resolve this by trusting the self-signed development certificate with the .NET CLI. To trust the certificate, run the following commands. First, remove the existing certificates.
+1. **Close all browser windows** and _try again_.
+1. If you're still experiencing the issue, then attempt to resolve this by trusting the self-signed development certificate with the .NET CLI. To trust the certificate, run the following commands. First, remove the existing certificates.
 
-> [!NOTE]
-> This will remove all existing development certificates on the local machine.
+  > [!NOTE]
+  > This will remove all existing development certificates on the local machine.
 
-```dotnetcli
-dotnet dev-certs https --clean
-```
+  ```dotnetcli
+  dotnet dev-certs https --clean
+  ```
 
-To trust the certificate:
+  To trust the certificate:
 
-```dotnetcli
-dotnet dev-certs https --trust
-```
+  ```dotnetcli
+  dotnet dev-certs https --trust
+  ```
```

Comment:
The alerts and code snippets aren't rendering correctly on the published page. You must indent to the first character numbered item's text to get it to actually appear under the item.
Correct way:
```markdown
1. ABC
...The ... at the start here show you that you must have 3 spaces to align properly.
```
Personally I use this to make sure it's 4 spaces indented on child content, which makes it easy to <kbd>TAB</kbd> position content:

```markdown
01. ABC
....This is four spaces in
```

This is what's on the published page:
![image](https://github.com/user-attachments/assets/b12e85bb-d184-4e42-bca8-90c670710745)